### PR TITLE
Added docs for client-side logger

### PR DESCRIPTION
**Pull request by tannakartikey**
### Summary

There is no mention of client-side ActionCable logger in [Action Cable Overview](https://edgeguides.rubyonrails.org/action_cable_overview.html) guide.

### Other Information
I added the entry inside the [Other Configuration](https://edgeguides.rubyonrails.org/action_cable_overview.html#other-configurations) section because I could not come up with any better place. Also, that section talks about the server side logging.

[Client-Side Components](https://edgeguides.rubyonrails.org/action_cable_overview.html#client-side-components) was another good place to add it but it did not make sense to add the whole section for such a small text.

### CODE REVIEW

File: guides/source/action_cable_overview.md [hunk 1]
```diff
@@ -771,6 +771,14 @@ config.action_cable.log_tags = [
 For a full list of all configuration options, see the
 `ActionCable::Server::Configuration` class.
 
+Client-side logs can be enabled with the following:
```
Comment (vipulnsward) [inline guides/source/action_cable_overview.md:774]: Lets move this to a new section called "### Client side logging" before `### Other Configurations`

```
### Client side logging

Client side logging is disabled by default. You can enable this by setting the `ActionCable.logger.enabled` to true.

...Code example here
```

File: actioncable/app/javascript/action_cable/logger.js [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import adapters from "./adapters"
 
+// The logger can be enabled with:
```
Comment (vipulnsward) [inline actioncable/app/javascript/action_cable/logger.js:3]:
```
//The logger is disabled by default. You can enable it with:
//
//   ActionCable.logger.enabled = true
//
```

Would be good to have an logging example here.

**Comment by tannakartikey** [comment]
@vipulnsward thanks for the review. I will implement those changes soon.

**Comment by tannakartikey** [comment]
@vipulnswardy I have made changes as per the review. Let me know if there is anything else. Thanks

**Pull request merged**